app/models/container_group.rb

```diff
@@ -11,7 +11,7 @@ class ContainerGroup < ActiveRecord::Base
   has_many :container_definitions, :dependent => :destroy
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
   has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
-  has_many :node_selector_parts, -> { where(:section => "node_selectors") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
+  has_many :selector_parts, -> { where(:section => "selectors") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
```

Comment:
Current installations having containerGroups with section => "node_selectors" will have to migrate to section => "selectors", does this make sense ? what is a node_selector and what is a selector ?